content/en/releases/release-managers.md

```diff
@@ -173,7 +173,6 @@ of each minor (1.Y) and patch (1.Y.Z) release
 GitHub team: [@kubernetes/build-admins](https://github.com/orgs/kubernetes/teams/build-admins)
 
 - Aaron Crickenberger ([@spiffxp](https://github.com/spiffxp))
-- Benjamin Elder ([@BenTheElder](https://github.com/BenTheElder))
 - Grant McCloskey ([@MushuEE](https://github.com/MushuEE))
 - Juan Escobar ([@juanfescobar](https://github.com/juanfescobar))
```

Comment:
actually, Juan is also not in anymore and @BenjaminKazemi is, let me just update all of those
/hold